### PR TITLE
Feat/alias/config ccs records

### PR DESCRIPTION
**Pull request by JeremyDec**
## Contact Center Solution Records
MANAGER-1854

### Description of the Change
New design of records page, for ccs numbers

Two commits : 
834ef2e => moving files
d1f60d5 => the new page

### CODE REVIEW

File: client/app/telecom/telephony/alias/records/translations/Messages_fr_FR.xml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<translations>
+   <translation id="telephony_alias_contactCenterSolution_records_title">Gestion des enregistrements</translation>
+   <translation id="telephony_alias_contactCenterSolution_records_subtitle">Retrouvez l’ensemble des appels enregistrés de votre file d’appels.</translation>
+   <translation id="telephony_alias_contactCenterSolution_records_subtitle_help">Ces appels sont automatiquement supprimés après 60 jours.</translation>
+
+   <translation id="telephony_alias_contactCenterSolution_records_activation_title">Activation de l’enregistrement</translation>
+   <translation id="telephony_alias_contactCenterSolution_records_activation">Enregistrement des appels</translation>
+   <translation id="telephony_alias_contactCenterSolution_records_activation_help">En activant cette option, vous acceptez <a href="{{link}}" rel="noopener" target="_blank">les conditions particulières</a> d’utilisation du service</translation>
```
[the remark this answers is not in the document]
Comment (FredericEspiau) [reply client/app/telecom/telephony/alias/records/translations/Messages_fr_FR.xml]: You meant "…"

[the remark this answers is not in the document]
Comment (marie-j) [reply client/app/telecom/telephony/alias/records/translations/Messages_fr_FR.xml]: No icon/span to indicate it opens a new tab ?

[the remark this answers is not in the document]
Comment (antleblanc) [reply client/app/telecom/telephony/alias/records/translations/Messages_fr_FR.xml]: It looks like a common issue according to how we manage translations.

I propose to found an appropriate solution in a separate task.

File: client/app/telecom/telephony/alias/records/telecom-telephony-alias-records.html [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+                        <span data-translate="telephony_alias_contactCenterSolution_records_enabled"></span>
+                    </oui-radio>
+                    <oui-radio data-value="false">
+                        <span data-translate="telephony_alias_contactCenterSolution_records_disabled"></span>
+                    </oui-radio>
+                </oui-radio-toggle-group>
+                <span class="oui-field__helper"
+                      data-translate="telephony_alias_contactCenterSolution_records_activation_help"
+                      data-translate-values="{ 'link': $ctrl.recordsGuideUrl }"
+                      data-translate-compile></span>
```
[the remark this answers is not in the document]
Comment (JeremyDec) [reply client/app/telecom/telephony/alias/records/telecom-telephony-alias-records.html]: You're right :)

**Pull request merged**